app/views/documents/fields/_govspeak.html.erb

```diff
@@ -1,3 +1,4 @@
-<%= render "govuk_publishing_components/components/govspeak", {} do %>
-  <%= raw Govspeak::Document.new(@document.contents[schema.id]).to_html %>
-<% end %>
+<%= render "components/markdown_raw", {
+  text: @document.contents[schema.id].to_s,
+  lines: 3
```

Comment:
This gives us the first 4 lines, which depending on how the editor (the person who's editing the text) is using empty lines can mean any of the following:
 - a title and a paragraph
 - a title a paragraph and a list item
 - only the first paragraph

Do you think we should skip counting the empty lines?

Reply:
The first and second are true, but I'm not sure how the lat one would happen unless they manually entered lots of newlines within the paragraph. I've tried it out with a few documents and think it looks alright (it'll probably never be perfect), but I could get Humin to look at it if you like?

Reply:
Yep, we can even go forward with this and test it with users.